packages/react-core/src/modules/slack/hooks/useManageSlackConnection.ts

```diff
@@ -0,0 +1,87 @@
+import { useKnockSlackClient } from "..";
+import { useCallback } from "react";
+
+const SLACK_AUTHORIZE_URL = "https://slack.com/oauth/v2/authorize";
+const TENANT_OBJECT_COLLECTION = "$tenants";
+const DEFAULT_SLACK_SCOPES = [
+  "chat:write",
+  "chat:write.public",
+  "channels:read",
+  "groups:read",
+  "groups:write",
+];
+
+type UseManageSlackConnectionOutput = {
+  buildSlackAuthUrl: () => string;
+  disconnectFromSlack: () => void;
+};
+
+function useManageSlackConnection(
```

Comment:
this was just moved and I removed the connection data from it that's now in the useSlackConnectionStatus hook